docs/dal/index.rst

```diff
@@ -319,54 +357,72 @@ datasets and/or access to additional data services.
 
 To obtain the names of the columns in a service response, write:
 
->>> print(resultset.fieldnames)
+.. doctest-skip::
+
+    >>> print(resultset.fieldnames)
 
 Rich metadata equivalent to what is found in VOTables (including unit,
 ucd, utype, and xtype) is available through resultset's
 :py:meth:`~pyvo.dal.query.DALResults.getdesc` method:
 
->>>  print(resultset.getdesc("accref").ucd)
+.. doctest-skip::
+
+    >>>  print(resultset.getdesc("accref").ucd)
 
 .. note::
     Two convenience functions let you retrieve columns of a specific
     physics (by UCD) or with a particular legacy data model annotation
     (by utype), like this:
 
+.. doctest-skip::
+
     >>> fieldname = resultset.fieldname_with_ucd('phot.mag;em.opt.V')
     >>> fieldname = resultset.fieldname_with_utype('Access.Reference')
 
 Iterating over a resultset gives the rows in the result:
 
->>> for row in resultset:
->>>     print row['accref']
-...
+.. doctest-skip::
+
+    >>> for row in resultset:
+    ...     print row['accref']
+    ...
 
 The total number of rows in the answer is available as its ``len()``:
 
->>> print(len(resultset))
-9
+.. doctest-skip::
+
+    >>> print(len(resultset))
+    9
 
 If the row contains datasets, they are exposed by several retrieval methods:
 
->>> url = row.getdataurl()
->>> fileobj = row.getdataset()
->>> obj = row.getdataobj()
+.. doctest-skip::
+
+    >>> url = row.getdataurl()
+    >>> fileobj = row.getdataset()
+    >>> obj = row.getdataobj()
```

Comment:
This last one is failing for me. Could point to a bug.

Reply:
opened https://github.com/astropy/pyvo/issues/324 for it, and commented out the line for now.